Source/ProgressionSystemRuntime/Public/Data/PSWorldSubsystem.h

```diff
@@ -18,6 +20,9 @@ class PROGRESSIONSYSTEMRUNTIME_API UPSWorldSubsystem : public UWorldSubsystem
 	static UPSWorldSubsystem& Get();
 	static UPSWorldSubsystem& Get(const UObject& WorldContextObject);
 
+	/* Delegate for informing row data changed */
+	FCurrentRowDataChanged OnCurrentRowDataChanged;
```

Comment:
Delegates are VERY useful for blueprint developers, please add `UPROPERTY(BlueprintAssignable, Category = "C++")` above delegate and use DYNAMIC_MULTICAST version of your macro instead of just DELEGATE

Reply:
done